lib/widgets/animated_qr/coconut_qr_data_handler.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
+
+import 'package:coconut_wallet/model/wallet/watch_only_wallet.dart';
+import 'package:coconut_wallet/utils/logger.dart';
+import 'package:coconut_wallet/widgets/animated_qr/i_coconut_qr_data_handler.dart';
+
+/// 코코넛 볼트 - 지갑 내보내기 스캔용
+class CoconutQRDataHandler implements ICoconutQrDataHandler {
```

Comment:
```suggestion
class CoconutQrDataHandler implements ICoconutQrDataHandler {
```

다른 클래스와 마찬가지로 Qr도 파스칼 케이스 어떤가요?

Reply:
feat/177-airgap-send-thirdparty 브랜치에서 수정하도록 하겠습니다~!